dbt_project.yml

```diff
@@ -2,7 +2,7 @@ name: "re_data"
 version: "0.10.4"
 config-version: 2
 
-require-dbt-version: [">=1.0.0", "<1.3.0"]
```

Comment:
@maciejklimek I think here we can do <1.4.0 to support patch releases (which shouldn't have any breaking changes) by default

Reply:
fixed